### PR TITLE
Ship remote page attachments

### PR DESCRIPTION
**Pull request by cramforce**
- Adds docs
- Adds validator rules
- Adds example
- Fixes robustness of drawer stylesheet (Ideally the UI would move into shadow DOM)

### CODE REVIEW

File: extensions/amp-story/validator-amp-story.protoascii [hunk 1]
```diff
@@ -771,6 +771,13 @@ tags: {  # <amp-story-page-attachment>
     value: "dark"
     value: "light"
   }
+  attrs: {
+    name: "href"
+    value_url: {
+      protocol: "http"
+      protocol: "https"
+    }
+  }
```
Comment (gmajoulet) [inline extensions/amp-story/validator-amp-story.protoascii:780]: Having any descendant today breaks the layout. I think we should have two separate validation specs for this `<amp-story-page-attachment>` tag:
1. The current one that allows descendants
2. A new one with a mandatory `href` attribute, and 0 allowed descendants

The alternative would be to remove all descendants at build time, but the validation spec is more strict.

File: extensions/amp-story/amp-story-page-attachment.md [hunk 2]
```diff
@@ -72,7 +72,19 @@ Values: "light" (default), "dark"
...
 
-## Allowed content and components
+## Linked content
+
+When providing a `href` attribute as page attachment, the respective URL is opened when the user activates the page attachment.
+
+```html
+<amp-story-page-attachment layout="nodisplay" href="https://www.example.com"
+  >...</amp-story-page-attachment
+>
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story-page-attachment.md:82]: Making it more clear that there is no children allowed:

```suggestion
<amp-story-page-attachment layout="nodisplay" href="https://www.example.com">
</amp-story-page-attachment>
```

File: extensions/amp-story/amp-story-page-attachment.md [hunk 3]
```diff
@@ -72,7 +72,19 @@ Values: "light" (default), "dark"
 >
 ```
 
-## Allowed content and components
+## Linked content
+
+When providing a `href` attribute as page attachment, the respective URL is opened when the user activates the page attachment.
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story-page-attachment.md:77]: Wdyt about adding one line about the fact that this is a page navigation on the web, but can be handled different in native contexts (in-app browser, customized app experience...)?

Comment (cramforce) [reply extensions/amp-story/amp-story-page-attachment.md:77]: I don't think we should specify what exactly happens.

File: extensions/amp-story/validator-amp-story.protoascii (hunk 1 above)
Comment (cramforce) [reply extensions/amp-story/validator-amp-story.protoascii:780]: Done.

Comment (gmajoulet) [reply extensions/amp-story/validator-amp-story.protoascii:780]: Can you push the changes?

Comment (cramforce) [reply extensions/amp-story/validator-amp-story.protoascii]: Sorry. Done

**Pull request merged**